.prospector.yaml

```diff
@@ -1,14 +1,58 @@
 output-format: grouped
-inherits:
-  - full_pep8
-  - strictness_low
+strictness: veryhigh
+test-warnings: true
+doc-warnings: true
+max-line-length: 79
 pep8:
-  options:
-    max-line-length: 79
+  full: true
 pylint:
   enable:
     - relative-import
+  disable:
+    - C0111  # Missing %s docstring
```

Comment:
Undecided about this one. We could fix so many of these in less than an hour of adding some docstrings.